.github/workflows/build.yml

```diff
@@ -161,30 +161,74 @@ jobs:
     - name: Copy confs
       run: |
         cp conf/default/* conf/
-    - name: Run services (2 mins)
-      env:
-        MYSQL_HOST: mysql
+    - name: Startup and character login checks
       run: |
-        ls -l
+        printf "\nStart server processes\n"
         screen -d -m -S topaz_connect ./topaz_connect --log login-server.log
         screen -d -m -S topaz_search ./topaz_search --log search-server.log
         screen -d -m -S topaz_game ./topaz_game --log map-server.log
-        sleep 2m
+        
+        printf "\nWaiting 15s for servers to fully start up\n"
+        sleep 15s
+        
```

Comment:
Is 15s enough time with the other processes to ensure the server has launched?  First launch of the day on my end usually takes around 60s+